CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ the detailed section referring to by linking pull requests or issues.
 
 - Implemented markdown for asset descriptions
 - Implemented list view for Broker UI
+- Asset Metadata is now editable
```

Comment:
Nit, it looks like the text is not fully displayed as the g is truncated

<img width="717" alt="image" src="https://github.com/sovity/edc-ui/assets/75735874/c5d3dbf2-166a-4d19-8a9d-1efb996cbf1f">

Reply:
The same happens with the text on top, right? At least in the my firefox UI I can see it cut off at the bottom at multiple places.

These are the styles of Angular Material, though, I've looked into it and it's multiple layers of overflow-hidden, no idea if we even have a chance to fight that.